Resources/Prototypes/Entities/Structures/Storage/Closets/wall_lockers.yml

```
@@ -153,6 +153,12 @@
       state_open: generic_open
       state_closed: mixed_door
 
+- type: entity
+  id: ClosetWallJanitor
+  parent: ClosetWallMixed
+  name: custodial closet
+  description: It's a storage unit for janitorial clothes and gear.
```

```suggestion
  description: It's a storage unit for janitorial supplies.
```
it doesn't have any clothes in it :p